src/main/java/uk/gov/hmcts/divorce/document/print/LetterPrinter.java

```diff
@@ -31,6 +36,11 @@ public void sendLetters(final CaseData caseData,
 
         List<Letter> letters = documentGenerator.generateDocuments(caseData, caseId, applicant, documentPackInfo);
 
+        var currentPacks = Optional.ofNullable(caseData.getDocuments().getLetterPacks()).orElse(new ArrayList<>());
```

Comment:
if you give it a default value of `new ArrayList()` in case documents you can avoid this Optional juggling